qa/tasks/deepsea.py

```diff
@@ -29,6 +28,21 @@ def anchored(log_message):
     return "{}{}".format(deepsea_ctx['log_anchor'], log_message)
 
 
+def remote_run_script_as_root(remote, path, data, args=None):
```

Comment:
I wonder if the usage of this method can be replaced by remote.sh_file?

Reply:
That's worth investigating as a future optimization. Since this PR has passed all tests and is needed to make the CI green, I'm merging it now.